src/window.js

```diff
@@ -1,3 +1,8 @@
-// assign alby-tools exports to global window object (for index.browser.js)
+// assign alby lightning exports to global window object (for index.browser.js)
+
+// @deprecated
 // @ts-ignore
-window["albyTools"] = require('./index.cjs');
\ No newline at end of file
+window["albyTools"] = require('./index.cjs');
+
+// @ts-ignore
+window["albyLightning"] = require('./index.cjs');
```

Comment:
we should use the new name here I guess.
window.lightningTools ? 

if the package is named lightning it's window.lightning ?

Reply:
why do we have those window variables in there at all? :thinking: 

for being able to use it via CDN includes?

Reply:
@reneaaron yes. e.g. using alby-tools in a simple codepen.